cura/BuildVolume.py

```diff
@@ -494,7 +494,7 @@ def _buildDisallowedAreaMesh(self, min_w: float, max_w: float, min_h: float, max
         mb = MeshBuilder()
         color = self._disallowed_area_color
         for polygon in self._disallowed_areas:
-            intersection = polygon.intersectionConvexHulls(bounding_box)
+            intersection = polygon.intersection(bounding_box)
```

Comment:
If we swap out `intersectionConvexHulls` for the normal `intersection` then we should also update our triangulation strategy